package-lock.json

```diff
@@ -1875,6 +1883,7 @@
       "version": "9.0.0",
       "resolved": "https://registry.npmjs.org/copy-webpack-plugin/-/copy-webpack-plugin-9.0.0.tgz",
       "integrity": "sha512-k8UB2jLIb1Jip2nZbCz83T/XfhfjX6mB1yLJNYKrpYi7FQimfOoFv/0//iT6HV1K8FwUB5yUbCcnpLebJXJTug==",
+      "dev": true,
```

Comment:
Only this line change is really caused by me, the rest of the file was adapted by `npm install`.